### PR TITLE
cherrypicked changes from 797696

### PR DESCRIPTION
**Pull request by DaveC49**
add user prefs section and link

### CODE REVIEW

File: help/C/Help_ch_Transactions.xml [hunk 1]
```diff
@@ -903,13 +903,20 @@ Author:
...
             of the main menu <guimenu>File</guimenu> entry, along with methods for importing
             other types of data. Select the appropriate entry in the <guisubmenu>Import</guisubmenu>
             sub menu to initiate importing transactions.
       </para>
     </sect2>
-
+    <sect2 id="trans-import-prefs">
+      <title>Import Preference</title>
+      <para>User preferences set in the <link linkend="prefs-online">Online Banking tab</link> of the user preferences
+        accessed by <menuchoice><guimenu>Edit</guimenu></menuchoice>-><menuchoice><guisubmenu>Preferences</guisubmenu></menuchoice> from the
```
Comment (fellen) [inline help/C/Help_ch_Transactions.xml:916]: `<menuchoice><guimenu>Edit</guimenu><guisubmenu>Preferences</guisubmenu></menuchoice>` generates a nicer output. ;-)

File: help/C/Help_ch_Transactions.xml [hunk 2]
```diff
@@ -903,13 +903,20 @@ Author:
         <listitem><para><emphasis role="bold"><link linkend="trans-import-MT940-MT942-DTAUS">DTAUS</link></emphasis>
              (.DTAUS)  Banking data Exchange File;</para></listitem>
       </itemizedlist>
+      <note><para>Files using these formats do not necessarily use the file extensions shown above.</para></note>
       <para>These import methods can be accessed from the <guisubmenu>Import</guisubmenu> submenu
             of the main menu <guimenu>File</guimenu> entry, along with methods for importing
```
Comment (fellen) [inline help/C/Help_ch_Transactions.xml:908]: I would prefer
`These import methods can be accessed from the <menuchoice><guimenu>File</guimenu><guisubmenu>Import</guisubmenu></menuchoice> submenu, ...`

Comment (DaveC49) [reply help/C/Help_ch_Transactions.xml]: Also done . Bob I'm a bit unsure of the procedure here once a PR has been created. If I just push these changes to my repo are they automatically then included in the PR?

File: help/C/Help_ch_Transactions.xml (hunk 1 above)
Comment (DaveC49) [reply help/C/Help_ch_Transactions.xml]: Done

**Comment by fellen** [comment]
Committed, thanks!

i took the liberty to remove "Bob" from the commit message as there was no Bob involved. So the hash changed from df3200a to 5be9c2a

File: help/C/Help_ch_Transactions.xml (hunk 2 above)
Comment (fellen) [reply help/C/Help_ch_Transactions.xml]: I am not Bob, I am Frank.

You did it right. You pushed your 2. commit and I pulled it then. The result are 2 commits in the history.

There is also another way:
Amend the first commit and force push your branch. Then the history would show one commit.

Comment (DaveC49) [reply help/C/Help_ch_Transactions.xml]: My apologies Frank.

**Comment by fellen** [comment]
I just watched a contradiction between the bug number 797696 (Add Xref to Online banking preferences setting in the Import transactions section) and this the serie #133, #134, #135, #136 of attempts. Unfortunately @gjanssens commented there.
(https://github.com/Gnucash/gnucash-docs/pull/133#issue-407649925) seems to be the right intro.
@DaveC49 can you enlighten us?

**Comment by DaveC49** [comment]
Frank  It was a major stuffup on my part after rebasing the original branch onto master instead on maint. The orignal comment is the right intro

"Minor edit to importing section of the transactions chapter to add a small section indicating that the online banking user preferences affect the import processing of files and provide a link to the Online banking tab section of the User Preferences setting under Customizing GnuCash.
Also adds a note to the effect that file extensions on import files are not necessarily present in import files."

David

**Pull request closed without merging**